rust/src/dcerpc/dcerpc.rs

```diff
@@ -357,6 +357,8 @@ impl DCERPCState {
             for tx_old in &mut self.transactions.range_mut(self.tx_index_completed..) {
                 index += 1;
                 if !tx_old.req_done || !tx_old.resp_done {
+                    tx_old.tx_data.updated_tc = true;
```

Comment:
why both sides here?

Reply:
We set both `tx_old.req_done = true;` and `tx_old.resp_done = true;`, so we modify both directions